run/src/main/openapi/run.yaml

```diff
@@ -753,7 +753,7 @@ components:
           description: the list of results
           items:
             type: object
-            additionalProperties: true
```

Comment:
I'm not sure how an `additionalProperties` to false behave when no specific properties are defined. Is it ok or does it prevent any content (which is probably not what we want)?